Application/EdFi.Ods.AdminApi.DBTests/ClaimSetEditorTests/DeleteClaimSetCommandTests.cs

```diff
@@ -26,15 +26,15 @@ public void ShouldDeleteClaimSet()
         {
             ApplicationName = $"Test Application {DateTime.Now:O}"
         };
-        Save(testApplication);
+        SaveAdminContext(testApplication);
 
```

Comment:
@jagudelo-gap 
I guess, no need for saving this application. Since, the claimset is not requiring one. Please remove all the SaveAdminContext.

Reply:
Done